core/src/main/java/com/netflix/conductor/core/events/EventQueues.java

```diff
@@ -55,24 +53,21 @@ private EventQueues() {
 				.collect(Collectors.toList());
 	}
 
-	public static ObservableQueue getQueue(String eventt, boolean throwException) {
-		String event = parametersUtils.replace(eventt).toString();
+	public static ObservableQueue getQueue(String eventType) {
+		String event = parametersUtils.replace(eventType).toString();
+		if (event.indexOf(':') == -1) {
```

Comment:
save event.indexOf to a variable and reuse

Reply:
done